crates/aptos-logger/src/aptos_logger.rs

```diff
@@ -33,15 +34,18 @@ use std::{
     thread,
 };
 use strum_macros::EnumString;
+use tokio::time;
 
 const RUST_LOG: &str = "RUST_LOG";
 const RUST_LOG_REMOTE: &str = "RUST_LOG_REMOTE";
-const RUST_LOG_TELEMETRY: &str = "RUST_LOG_TELEMETRY";
+pub const RUST_LOG_TELEMETRY: &str = "RUST_LOG_TELEMETRY";
 const RUST_LOG_FORMAT: &str = "RUST_LOG_FORMAT";
 /// Default size of log write channel, if the channel is full, logs will be dropped
 pub const CHANNEL_SIZE: usize = 10000;
 const NUM_SEND_RETRIES: u8 = 1;
 const FLUSH_TIMEOUT: Duration = Duration::from_secs(5);
+const FILTER_REFRESH_INTERVAL: Duration =
+    Duration::from_secs(5 /* minutes */ * 60 /* seconds */);
```

Comment:
@sitalkedia I have reduced the update cadence to 5 minutes. With 30 minute frequency, we may not get the logs when we need them. Let me know if you want to decrease the frequency for some reason.

Reply:
I think it is okay to start with 5 mins refresh interval for this.